user-manual/metadata/scripts/METS.xml

```diff
@@ -0,0 +1,8984 @@
+            <premis:agentIdentifier>
+              <premis:agentIdentifierType>Archivematica user pk</premis:agentIdentifierType>
+              <premis:agentIdentifierValue>1</premis:agentIdentifierValue>
+            </premis:agentIdentifier>
+            <premis:agentName>username="analyst", first_name="", last_name=""</premis:agentName>
+            <premis:agentType>Archivematica user</premis:agentType>
+          </premis:agent>
+        </mets:xmlData>
+      </mets:mdWrap>
+    </mets:digiprovMD>
+  </mets:amdSec>
+  <mets:amdSec ID="amdSec_2">
+    <mets:techMD ID="techMD_2">
+      <mets:mdWrap MDTYPE="PREMIS:OBJECT">
+        <mets:xmlData>
+          <premis:object xmlns:premis="http://www.loc.gov/premis/v3" xsi:type="premis:file" xsi:schemaLocation="http://www.loc.gov/premis/v3 http://www.loc.gov/standards/premis/v3/premis.xsd" version="3.0">
+            <premis:objectIdentifier>
+              <premis:objectIdentifierType>UUID</premis:objectIdentifierType>
+              <premis:objectIdentifierValue>d0c46bbb-63b1-4530-ad1f-f65d9a32e434</premis:objectIdentifierValue>
+            </premis:objectIdentifier>
+            <premis:objectIdentifier>
+              <premis:objectIdentifierType>ULID</premis:objectIdentifierType>
+              <premis:objectIdentifierValue>01D9T8NH7QF5WW5CJ4KTW15J3C</premis:objectIdentifierValue>
+            </premis:objectIdentifier>
+            <premis:objectIdentifier>
+              <premis:objectIdentifierType>EXID</premis:objectIdentifierType>
+              <premis:objectIdentifierValue>https://example.com/file/id/292468806</premis:objectIdentifierValue>
+            </premis:objectIdentifier>
+            <premis:objectCharacteristics>
+              <premis:compositionLevel>0</premis:compositionLevel>
+              <premis:fixity>
+                <premis:messageDigestAlgorithm>sha256</premis:messageDigestAlgorithm>
+                <premis:messageDigest>383d349019ace0e235443c6cb8c5fa3174f00d562281947d36f5fd12aa263687</premis:messageDigest>
+              </premis:fixity>
+              <premis:size>6271469</premis:size>
+              <premis:format>
+                <premis:formatDesignation>
+                  <premis:formatName>Raw JPEG Stream</premis:formatName>
+                  <premis:formatVersion></premis:formatVersion>
+                </premis:formatDesignation>
+                <premis:formatRegistry>
+                  <premis:formatRegistryName>PRONOM</premis:formatRegistryName>
+                  <premis:formatRegistryKey>fmt/41</premis:formatRegistryKey>
+                </premis:formatRegistry>
+              </premis:format>
+              <premis:creatingApplication>
+                <premis:dateCreatedByApplication>2019-05-23</premis:dateCreatedByApplication>
+              </premis:creatingApplication>
+              <premis:objectCharacteristicsExtension>
```

Comment:
There are a lot of objectCharacteristicsExtension containers in this sample file with some very verbose content. If you'd like to reduce the verbosity of the file, you might want to leave this first objectCharacteristicsExtension container as is but replace the content of all the others with something like `<!--Content removed for the sake of brevity -->`.

Reply:
Very good idea. And having done it, I see what you mean about removing verbose content! Thank you.